dist/info/squirreljme_libretro.info

```diff
@@ -41,9 +41,9 @@ supports_no_game = "true"
 
 # The number of mandatory/optional firmware files the core needs:
 firmware_count = 1
-firmware0_desc = "squirreljme.sqc (SquirrelJME Compiled SummerCoat ROM)"
+firmware0_desc = "squirreljme.sqc (SquirrelJME ROM)"
 firmware0_path = "squirreljme.sqc"
-firmware0_opt = "false"
+firmware0_opt = "true"
 
 # Additional notes:
 #notes = "(!) hash|(!) game rom|(^) continue|[1] notes|[^] continue|[*] list"
```

Comment:
Is this the right MD5?

``` suggestion
notes = "(!) squirreljme.sqc (md5): 8e98b0c8e5c96c191a8b2a9d29d0454b|(!) Homepage: https://squirreljme.cc"
```

Reply:
Would not have an MD5 for the ROM and accept anything since they are generated per-commit, since the ROM just contains the implementation of the Java Runtime.